api/src/pcapi/routes/backoffice/collective_offers/collective_offers_blueprint.py

```diff
@@ -469,12 +442,17 @@ def edit_collective_offer_price(collective_offer_id: int) -> utils.BackofficeRes
             return redirect(redirect_url, code=303)
 
         try:
-            if feature.FeatureToggle.PRICE_FINANCE_EVENTS.is_active():
-                finance_api.cancel_latest_event(collective_booking)
-            else:
-                finance_api.cancel_pricing(
-                    booking=collective_booking, reason=finance_models.PricingLogReason.CHANGE_AMOUNT
+            cancelled_event = finance_api.cancel_latest_event(collective_booking)
+            if cancelled_event:
+                finance_api.delete_pricing_point_pricings_from_date(
+                    cancelled_event.pricingPointId, cancelled_event.pricingOrderingDate, commit=False
```

Comment:
Après réflexion, on n'a pas besoin de faire ça pour les réservations collectives, car leur montant n'est pas pris en compte dans le chiffre d'affaires. Donc leur inclusion (ou pas) et leur montant (ou changement de montant) n'impacte pas le calcul du remboursement des autres réservations (individuelles ou collectives). 

```suggestion
```

L'annulation du FinanceEvent et la création d'un nouveau FinanceEvent suffisent. Désolé pour la mauvaise piste que j'ai donné en commentaire de ta précédente pull request. Ceci dit, la fonction `delete_pricing_point_pricings_from_date()` pourrait servir de base à la correction sur laquelle travaille @Vencespass . (Mais je ne l'ai pas relue, et il serait préférable de ne pas merger cette fonction dans ta PR.)

Reply:
D'acc c'est noté ! J'aurais dû y penser j'étais déjà passé sur ce calcul du new_revenue :D